ci/pipeline.yaml

```diff
@@ -111,17 +130,37 @@ jobs:
             tag: version/version
             globs:
               - build/*
-        - put: docker-image
-          inputs:
-            - build
-            - source
-            - version
-          params:
-            build: .
-            dockerfile: source/Dockerfile
-            tag_as_latest: false
-            tag_file: version/version
-
+        - do:
+          - get: oci-build-task
+          - get: ubuntu-image
+            params:
+              format: oci
+          - file: version/version
+            load_var: image_version
+          - task: build-oci
+            privileged: true
+            image: oci-build-task
+            config:
+              platform: linux
+              inputs:
+                - name: source
+                - name: build
+                - name: ubuntu-image
+              outputs:
+                - name: image
+              params:
+                CONTEXT: .
+                DOCKERFILE: source/Dockerfile
+                IMAGE_ARG_base_image: ubuntu-image/image.tar
+              run:
+                path: build
```

Comment:
Similiar to https://github.com/cf-platform-eng/needs/pull/340 , is there a path conflict with `build` input?  may need some input mapping or refactor the `source/ci/tasks/build.yml` task

Reply:
I was under the impression that https://github.com/cf-platform-eng/needs/blob/main/ci/tasks/build.yml is actually importing source/build so it doesn't conflict

Reply:
https://github.com/cf-platform-eng/tileinspect/blob/8e16d66dcc9a369eda9fb277a36086e9b8b61736/ci/tasks/build.yml#L18

Reply:
I'll update these tasks to avoid the collision

Reply:
I don't believe this is a conflict based on this run of the `needs` pipeline https://tpe-concourse-rock.acc.broadcom.net/teams/ppe-isv/pipelines/needs/jobs/build/builds/19, however given the readability issues alone I'll make these names distinct. Thank you